be/src/storage/version_graph.h

```diff
@@ -64,14 +64,18 @@ class VersionGraph {
 
 private:
     /// Private method add a version to graph.
-    std::unique_ptr<Vertex>& _add_vertex_to_graph(int64_t vertex_value);
+    int64_t _add_vertex_to_graph(int64_t vertex_value);
 
     // OLAP version contains two parts, [start_version, end_version]. In order
     // to construct graph, the OLAP version has two corresponding vertex, one
     // vertex's value is version.start_version, the other is
     // version.end_version + 1.
     // Use adjacency list to describe version graph.
-    std::unordered_map<int64_t, std::unique_ptr<Vertex>> _version_graph;
+    std::vector<Vertex> _version_graph;
```

Comment:
When can the vertexes be removed from this graph?
If they aren't removed, it will cause a memory leak.

Reply:
only release memory after reconstruct version graph. Seems to be a potential memory leak from older version.